apps/client/src/utils/keycloak/init.js

```diff
@@ -27,6 +27,7 @@ export const getUserProfile = async () => {
 
 export const keycloakInit = async () => {
   try {
+    keycloakConf.redirectUri = location
```

Comment:
Peut-être plutôt modifier la valeur de `redirectUri` dans le fichier `keycloak/config.js` ?

Reply:
d'où vient la valeur de location?

Reply:
@baudointran du navigateur, il s'agit de `Window.location` : https://developer.mozilla.org/fr/docs/Web/API/window/location

Reply:
@this-is-tobi j'ai testé, en fait j'ai l'impression que `location` n'est pas accessible depuis le `config.js` :

```
dso-console_client  | error when starting dev server:
dso-console_client  | ReferenceError: location is not defined
```

Quand je fais un `console.log({ Window }) j'ai une erreur de type `ReferenceError: Window is not defined`, alors que je peux le faire sans problème dans `init.js`.

Je ne sais pas ce qui fait que l'un à accès au Window et pas l'autre